src/ui_fsmenu/multiplayersetupgroup.cc

```diff
@@ -265,8 +265,9 @@ struct MultiPlayerPlayerGroup : public UI::Box {
 			n->set_player_state(id_, state);
 
 			const GameSettings& settings = settings_->settings();
-			if (state == PlayerSettings::State::kComputer &&
-			    !settings.get_tribeinfo(settings.players[id_].tribe).suited_for_ai) {
+			const std::string& tribe = settings.players[id_].tribe;
+			if (state == PlayerSettings::State::kComputer && tribe.empty() &&
+			    !settings.get_tribeinfo(tribe).suited_for_ai) {
```

Comment:
This looks suspicious: `settings.get_tribeinfo` will only get empty strings. 
Maybe you wanted `!tribe.empty()` ?

Reply:
Ups, good catch